packages/search/src/search.tsx

```diff
@@ -0,0 +1,422 @@
+/**
+ * External dependencies
+ */
+import React, { KeyboardEvent, MouseEvent, FocusEvent, FormEvent, ChangeEvent } from 'react';
```

Comment:
Opinionated nit: would be really great if we can sort the imports, both when they're multiple imports from the same module and multiple imports on multiple lines. Makes it more readable IMHO.

Reply:
Shall we enable an ESLint rule for ordered imports?

Reply:
My personal opinion is YES, but maybe now it's not the time. With the new rule that we recently enabled to make all imports relative to the root of Calypso, any change to existing files will be a mess for contributors.

I say let's reconsider doing at some point in the future.

Reply:
/cc @scinos

Reply:
Plenty of options if we decide go that route https://www.npmjs.com/search?q=eslint%20sort%20import